src/pages/LOCsPage.js

```diff
@@ -17,7 +18,7 @@ const mapDispatchToProps = (dispatch) => ({
 class LOCsPage extends Component {
 
     componentWillMount(){
-        if (this.props.isNeedReload) {
+        if (!this.props.isReady && !this.props.isFetching) {
             this.props.getLOCs(localStorage.chronoBankAccount);
```

Comment:
you don't need to pass _from_ parameter for _call_ functions, so you can remove a lot of such code, for instance in LOCDAO